packages/preview-middleware-client/package.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "@sap-ux-private/preview-middleware-client",
+    "version": "0.0.1",
+    "description": "Client-side coding hosted by the preview middleware",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/preview-middleware-client"
+    },
+    "license": "Apache-2.0",
```

```suggestion
    "license": "SEE LICENSE IN LICENSE",
```
I believe this is recommended approach and adding the `"license": "LICENSE` in the root of the module.

---

We use `"license": "Apache-2.0"` in all other modules in the repository, why would we do it different for this one?

---

We seem to have a mix as of now. Lets discuss separate to this PR and then update to align all modules.